proter/src/main/scala/com/workflowfm/proter/flows/Flow.scala

```diff
@@ -52,19 +52,22 @@ class NoTask() extends Flow { // this can't be a case class/object because of th
 
 class FlowTask(t: Task) extends Flow {
   override val id: UUID = t.id.getOrElse(UUID.randomUUID)
+  override def toString(): String = t.name
   val task: Task = t.withID(id) // make sure the task has an ID
 
   override def copy(): Flow = new FlowTask(task.withID(UUID.randomUUID))
 }
 
 class Then(val left: Flow, val right: Flow) extends Flow {
   override def copy(): Flow = new Then(left.copy(), right.copy())
+  override def toString(): String = "(" + left.toString + " > " + right.toString + ")"
 
   override def >(f: Flow): Then = new Then(left, right > f) // make > right associative
 }
 
 class And(val left: Flow, val right: Flow) extends Flow {
   override def copy(): Flow = new And(left.copy(), right.copy())
+  override def toString(): String = "(" + left.toString + " + " + right.toString + ")"
```

Comment:
This is ok for us, but you would normally want to split changes to different functionality (`toString` vs. `RandomFlowFactory`) in different commits.